contracts/BroadcastMarket.sol

```diff
@@ -166,6 +209,36 @@ contract BroadcastMarket is Ownable, ReentrancyGuard {
             });
     }
 
+    function getNCoords() public view returns (uint256) {
+        return coordIds.length;
+    }
+
+    function getCoords(uint256 location) public view returns (Coord memory) {
+        Coord memory coord = coords[location];
+        require(coord.x != 0 && coord.y != 0 || coord.isOrigin, "No Coord for that Planet");
```

Comment:
I think this is right? tests pass

Reply:
Ah, I see how you were doing this. I was definitely thinking `isOrigin` would be a flag like `wasSubmitted` since the boolean default would be false, then we wouldn't need the x/y checks at all, right?

Reply:
I dont know what wasSubmitted is and dont understand the question

Reply:
Like, instead of having this special case for `isOrigin`, just set a flag on all Coords structs that you check to know if it is a "defaulted" one or if it is actually created in the system.

Reply:
oh you mean change `isOrigin` to something like `isInitialized`

Reply:
Yeah! I just didn't want to use `isInitialized` in the naming because `initialize` means something specific to the DF contracts.